src/style.css

```diff
@@ -1 +1,79 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Book.ttf');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Heavy.ttf');
+  font-weight: 900;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.search-group {
+  display: flex;
+  flex-direction: column;
+  gap: 20px; /* Espaçamento de 20px entre os elementos filhos */
+  padding: 20px 8px;
+}
+
+.search__bar {
+  font-size: 16px;
+  box-sizing: border-box;
+  border: 1px solid rgba(225, 231, 237, 1);
+  width: 100%; /* Ajuste para largura total */
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  font-family: Avenir, Arial, sans-serif;
+}
+
+.search__bar::placeholder {
+  color: #3d4e61;
+}
+
+.search__bar--top {
+  height: 70px;
+  border-radius: 4px 4px 0 0;
+  border: 1px solid rgba(225, 231, 237, 1);
+  opacity: 1;
+  padding: 19px 10px 19px 62px; /* Ajuste o padding esquerdo para 62px */
+  background-image: url('./images/Search.svg');
+  background-repeat: no-repeat;
+  background-position: 26px center;
+  background-size: 19px 19px;
+}
+
+.search__bar--bottom {
+  height: 42px;
+  border-radius: 0 0 4px 4px;
+  border: 1px solid rgba(225, 231, 237, 1);
+  opacity: 1;
+  padding: 11px 10px 11px 33px;
+  background-image: url('./images/Search.svg');
+  background-repeat: no-repeat;
+  background-position: 13px center; /* Ícone a 11px da esquerda */
+  background-size: 11px 11px;
+  font-size: 14px;
+}
+
+.search__bar:focus {
+  border-color: rgba(0, 0, 0, 0.25);
+  outline: none;
+  box-shadow: 0 0 5px rgba(0, 0, 0, 0.25);
+  font-family: Avenir-Book, Arial, sans-serif; /* Usar Avenir-Heavy no focus */
+  font-weight: 900;
+  text-shadow: 0 4px 4px #00000040;
```

Comment:
The font-family 'Avenir-Book' used here does not match the font-family 'Avenir' defined in the @font-face rule. Instead, you should specify the font-weight to ensure the correct font is used. Also, 'font-weight: 900;' is incorrect here because 'Avenir-Book' is a lighter weight (likely 300), not the heaviest weight (900).